parm/gdas/aero_crtm_coeff.yaml

```diff
@@ -1,13 +1,13 @@
 mkdir:
 - $(DATA)/crtm/
 copy:
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/AerosolCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/CloudCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_npp.SpcCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_npp.TauCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_j1.SpcCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/v.viirs-m_j1.TauCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISice.EmisCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISland.EmisCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISsnow.EmisCoeff.bin, $(DATA)/crtm/]
-- [$(FV3JEDI_FIX)/crtm/$(crtm_VERSION)/NPOESS.VISwater.EmisCoeff.bin, $(DATA)/crtm/]
+- [$(CRTM_FIX)/AerosolCoeff.bin, $(DATA)/crtm/]
+- [$(CRTM_FIX)/CloudCoeff.bin, $(DATA)/crtm/]
```

Comment:
Can you remind us where is `CRTM_FIX` defined?

Reply:
for GSI, it's in the CRTM module file. I expect this to be the 'official' way in the future. For now, this is defined in the GDASApp modulefile and points to a directory I maintain. This is because of big vs little endian for the coefficients... GSI wants one and JEDI the other. CRTMv3 with netCDF files will make this problem go away.

Reply:
Thank you.